run-sonar-scanner.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -eo pipefail
+
+if [[ "$RUNNER_OS" == "Windows" ]]; then
+  SCANNER_BIN="sonar-scanner.bat"
+else
+  SCANNER_BIN="sonar-scanner"
+fi
+
+scanner_args=()
+if [[ ${SONARCLOUD_URL} ]]; then
+  scanner_args+=("-Dsonar.scanner.sonarcloudUrl=${SONARCLOUD_URL}")
+fi
+
+if [[ "$RUNNER_DEBUG" == '1' ]]; then
+  scanner_args+=('--debug')
+fi
+
+if [[ -n "${INPUT_PROJECTBASEDIR}" ]]; then
+  scanner_args+=("-Dsonar.projectBaseDir=${INPUT_PROJECTBASEDIR}")
+fi
+
+if [[ -n "${SONAR_ROOT_CERT}" ]]; then
```

Comment:
Related to certificates: `sonar-scanner-cli-docker` supports [this deprecated feature](https://github.com/SonarSource/sonar-scanner-cli-docker/blob/master/bin/entrypoint.sh#L13), where it copies certificates in `/tmp/cacerts` into the `$SONAR_USER_HOME/ssl/truststore.p12`, for backward compatibility.
This is not supported anymore.

Reply:
Maybe we should reintroduce it, and deprecate it in a following version of the action, to avoid mixing concern of release 4.0.0. WDYT @henryju ?

Reply:
> where it copies certificates in /tmp/cacerts

This is `/tmp/cacerts` inside the container. The folder on the host is not guaranteed to be named the same. So I would take the opportunity of a major version upgrade to drop this.